packages/documents/graphql/schema-types.js

```diff
@@ -42,6 +53,7 @@ type Meta {
   # the id of the discussion itself
   discussionId: ID
   credits: JSON
+  audioSources: [AudioSource]
```

Comment:
This is going to be interesting... if the publikator writes something to `audioSources` array that doesn't fit the `AudioSource` spec, apollo is going to explode on delivery of the document. But I think this fine.